CONTRIBUTING.rst

```diff
@@ -62,8 +62,25 @@ but you can also use a ``virtualenv`` of your own creation.
 
 .. _repo: https://github.com/GoogleCloudPlatform/google-cloud-python
 
-Using a custom ``virtualenv``
-=============================
+Use ``nox``, not ``virtualenv``
+===========================
+
+- To install dependencies into a virtual environment, run the unit test via nox.
```

Comment:
Needs a link to the [nox docs](https://nox.readthedocs.io/en/latest/).

Reply:
I've often considered making Nox's woodcut a fox and giving it the HTML title attribute of "The Nox Docs Fox".